apps/src/storage/dataBrowser/dataVisualizer/Snapshot.jsx

```diff
@@ -136,7 +136,7 @@ class Snapshot extends React.Component {
         >
           <div ref="snapshot">
             <h1>{this.props.chartTitle}</h1>
-            <img src={this.state.imageSrc} />
+            <img style={{maxHeight: '50vh'}} src={this.state.imageSrc} />
```

Comment:
is there a difference between `50%` and `50vh` in this case? if not, we usually use percentages and should update for consistency

Reply:
There is, `50vh` is 50% of the window size, `50%` is 50% of the max size of the parent div. In this case the parent div doesn't have a max size, so setting `50%` wouldn't do anything.